WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -270,7 +270,13 @@
     <Panel IsVisible="{Binding IsTransactionHistoryEmpty}">
       <c:HistoryPlaceholderPanel Margin="{StaticResource HistoryTablePlaceholderPanelMargin}"
                                  RowHeight="{StaticResource HistoryTableRowHeight}" />
-      <Panel Background="{DynamicResource RegionColor}" Opacity="0.4" />
+      <Panel Background="{DynamicResource RegionColor}" Opacity="0.75" />
+      <Grid RowDefinitions="26,1*,1.5*" IsVisible="{Binding IsTransactionHistoryLoaded}">
+        <c:InfoMessage VerticalAlignment="Center"
+                       HorizontalAlignment="Center"
+                       Grid.Row="1"
+                       Content="Transaction history is empty" />
+      </Grid>
```

Comment:
@turbolay

I have pushed simplifications so the actual change is just a few lines of code.
For this part, I don't see how the `Grid` could be avoided nicely with keeping the behavior you implemented.
So ok go with it.

The only questionable part is the first row of the Grid which is 26 high. On the smaller resolution of the Window, the message intersects with the header row. It should be under that. Can you adjust it?

Reply:
> The only questionable part is the first row of the Grid which is 26 high. On the smaller resolution of the Window, the message intersects with the header row. It should be under that. Can you adjust it?

I broke it, pushed fix.